chirp/drivers/ft1d.py

```diff
@@ -1131,7 +1132,8 @@ def _get_aprs_beacons(self):
                 checksum = body[-2:]
                 body = ''.join(s for s in body[:-2]
                                if s in string.printable).translate(
-                                   None, "\x09\x0a\x0b\x0c\x0d")
+                                   str.maketrans(
+                                       "", "", "\x09\x0a\x0b\x0c\x0d"))
```

Comment:
Wasn't this the driver we had trouble with loading the memory name from the image? Did you figure out what was going on with that?